User defined dataset bounds (#596)
NASA-IMPACT/veda-ui: docs/content/frontmatter/layer.md

  `string`  
  The colormap to use for the layer. One of https://cogeotiff.github.io/rio-tiler/colormap/#default-rio-tilers-colormaps

**bounds**  
`[int, int, int, int] | fn(bag)`  

> Initial bounds for the map. This is useful for datasets that are not global, and for which the STAC bounds are not appropriate.

Reviewer: ```suggestion
Initial bounds for the map. This is useful for adjusting the initial view on datasets for which the STAC bounds are not appropriate.
```
